examples/tutorials/tutorial_BET_Disk.py

```diff
@@ -0,0 +1,159 @@
+import json
+
+import flow360 as fl
+import flow360.component.simulation.units as u
+from flow360.component.simulation.unit_system import SI_unit_system
+from flow360.examples import TutorialBETDisk
+
+"""
+In this tutorial case we are using Blade Element Theory (BET) to simulate the XV-15 rotor blade. Documentation for this tutorial is available in the link below.
+
+https://docs.flexcompute.com/projects/flow360/en/latest/tutorials/BETTutorial/BETTutorial.html
+"""
+
+
+fl.Env.preprod.active()
+
+project = fl.Project.from_file(TutorialBETDisk.geometry, name="Tutorial BETDisk from Python")
+geometry = project.geometry
+
+# show face and edge groupings
+geometry.show_available_groupings(verbose_mode=True)
+geometry.group_faces_by_tag("faceName")
+geometry.group_edges_by_tag("edgeName")
+
+
+bet = json.loads(open(TutorialBETDisk.extra["disk0"]).read())
```

Comment:
Could you also describe how is the disk0.json file generated? Maybe you can describe in the docstring that for an existing user with the old Flow360.json file, they can use the conversion function you wrote to get the BETDisk info for the new Python API. Also mention where the conversion function is and how to use it.

Reply:
I see, then we don't call the conversion function here and keep using disk0 file as is.